.env.example

```diff
@@ -5,4 +5,6 @@ ALCHEMY_MAINNET_RPC_URL="https://eth-mainnet.alchemyapi.io/v2/your-api-key"
 REPORT_GAS=true
 COINMARKETCAP_API_KEY="YOUR_KEY"
 AUTO_FUND=true
-VRF_SUBSCRIPTION_ID=YOUR_SUBSCRIPTION_ID
\ No newline at end of file
+VRF_SUBSCRIPTION_ID=YOUR_SUBSCRIPTION_ID
```

Comment:
I personally prefer this in the `hardhat.config.js` but if you like it here that's cool.